audit-ci.jsonc

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
If this would be the default branch, then security audit would be handled directly by GitHub.

This configuration needs to be only inside the branch that would be scanned via `audit-ci`